nipype/interfaces/ants/resampling.py

```diff
@@ -246,9 +246,13 @@ class ApplyTransformsInputSpec(ANTSCommandInputSpec):
     interpolation_parameters = traits.Either(traits.Tuple(traits.Int()),  # BSpline (order)
                                              traits.Tuple(traits.Float(),  # Gaussian/MultiLabel (sigma, alpha)
                                                           traits.Float())
-                                             )
-    transforms = InputMultiPath(
-        File(exists=True), argstr='%s', mandatory=True, desc='transform files: will be applied in reverse order. For example, the last specified transform will be applied first')
+                                            )
+    transforms = traits.Either(InputMultiPath(File(exists=True), desc='transform files: will be '
```

Comment:
If you don't pass a --transforms option to antsApplyTransforms it uses the identity. Would this be a better option?